spec/core/elastic_connector_actions_spec.rb

```diff
@@ -2,9 +2,15 @@
 require 'connectors/connector_status'
 require 'connectors/sync_status'
 require 'utility'
+require './spec/support/filter_validation_result_helper'
+
+RSpec.configure do |c|
+  c.include FilterValidationResultHelper
+end
```

Comment:
This kind of thing is more common to put in `spec/spec_helper.rb`. However, it looks like this is only used in this spec file - why not define that `filter_validation_result` function directly in this file, rather than in a separate module that you then have to include?

Reply:
Makes sense, just saw an article, that these should be included in external modules to have a clear separation. But in this case it's probably correct to include it in the RSpec as it's not used outside of it.